tests/testHtmIndex.py

```diff
@@ -176,15 +179,20 @@ def testSanity(self):
         self.assertGreater(numWithSources, 0)
 
     def testAgainstPersisted(self):
-        pix_id = 671901
+        pix_id = 2222
         dataset_name = IngestIndexedReferenceTask.ConfigClass().dataset_config.ref_dataset_name
         data_id = self.indexer.make_data_id(pix_id, dataset_name)
         self.assertTrue(self.test_butler.datasetExists('ref_cat', data_id))
         ref_cat = self.test_butler.get('ref_cat', data_id)
+        if REGENERATE_COMPARISON:
+            os.unlink(self.test_cat_path)
+            ref_cat.writeFits(self.test_cat_path)
+            return
```

Comment:
Please make it clear that `REGENERATE_COMPARISON` is true and the test will not be run. My suggestion is to call `self.fail` instead of returning, so it is impossible to accidentally leave this set.